melodic/distribution.yaml

```diff
@@ -832,6 +832,13 @@ repositories:
       url: https://github.com/fetchrobotics/fetch_msgs.git
       version: melodic-devel
     status: maintained
+  fetch_open_auto_dock:
+    release:
```

Comment:
We recommend adding source and doc entries to make it easier for users to build from source as well as getting documentation content generated for the wiki pages.
You can refer to the other fetch-robotics packages for an example: 

https://github.com/ros/rosdistro/blob/f3336ee116a09e976ae7d82226590c1aad912b1c/melodic/distribution.yaml#L816-L833